### ISSUE TITLE
sqlitebrowser not building

### ISSUE THREAD

**Issue by sjau** [NixOS/nixpkgs#42320]
## Issue description

Not really sure if it's becuase of qt5_use_modules or not. At least CMake errors there.

````
building '/nix/store/p9fdv8p15r0848ryk784bx6zvrgl3c1h-unit-avahi-daemon.socket.drv'...                                                                                                                                                       
building '/nix/store/8pix1r6g1ahphz72r92cpj0kp2xzyaiy-stage-1-init.sh.drv'...                                                                                                                                                                
building '/nix/store/j41shwdsgw0aqx2pm0milk9hqmldi561-unit-bluetooth.service.drv'...                                                                                                                                                         
building '/nix/store/1fyn2dyv1cmpqkvnilqgz6gmp6zf4sb1-unit-console-getty.service-disabled.drv'...                                                                                                                                            
building '/nix/store/8k2qnsk66gk3wjx5qpkfdnhz5wxm8vs3-initrd.drv'...                                                                                                                                                                         
building '/nix/store/zwax2zqx982sf1iv0sh67kaq2gb1y4ll-unit-container-getty-.service.drv'...                                                                                                                                                  
-- Detecting C compile features - done                                                                                                                                                                                                       
-- Check for working CXX compiler: /nix/store/cg650dmmsy2bmsbx1y5f8qqpjb5hr0g2-gcc-wrapper-7.3.0/bin/g++                                                                                                                                     
unpacking sources                                                                                                                                                                                                                            
unpacking source archive /nix/store/xx1h9j35i6xyivl85fn44fvzh9aivi0x-libreoffice-6.0.4.2.tar.xz                                                                                                                                              
building '/nix/store/66clb9kdp484hszxfbmqmnam9msry3kp-unit-cups-browsed.service.drv'...                                                                                                                                                      
-- Check for working CXX compiler: /nix/store/cg650dmmsy2bmsbx1y5f8qqpjb5hr0g2-gcc-wrapper-7.3.0/bin/g++ -- works                                                                                                                            
-- Detecting CXX compiler ABI info                                                                                                                                                                                                           
building '/nix/store/5aqsrhg8zaha86f940ghvks7mi4xphw5-unit-dbus.socket.drv'...                                                                                                                                                               
building '/nix/store/a28krzfpf2wyf1qyxklvlq8rxqscp177-unit-firewall.service.drv'...                                                                                                                                                          
-- Detecting CXX compiler ABI info - done                                                                                                                                                                                                    
building '/nix/store/mv2kwm1za55zf6w07hh5j93151dk6jxh-unit-fs.target.drv'...                                                                                                                                                                 
-- Detecting CXX compile features                                                                                                                                                                                                            
building '/nix/store/j6k9dnwl8ws5g6985q61r19dbaxhmicd-unit-getty-.service.drv'...                                                                                                                                                            
building '/nix/store/yw1111nvn3zlsr2yay32f1a71rfi0v40-unit-graphical-session.target.drv'...                                                                                                                                                  
building '/nix/store/wz295xf2n0rbpyc0y4y03qx9zk7km32p-unit-keys.target.drv'...                                                                                                                                                               
building '/nix/store/fcqv7xqg5pfngq3if056riqbx3awn3gq-unit-libvirt-guests.service.drv'...                                                                                                                                                    
building '/nix/store/dc8zb1g9hlrsrm8s8q0b0z27swv1bhwg-unit-local-fs.target.drv'...                                                                                                                                                           
building '/nix/store/ij5457m7rfdpqllp1gwpk6pnhrbd9bmc-unit-mdadm-grow-continue-.service.drv'...                                                                                                                                              
building '/nix/store/sbr8fgqzkg5d7jy2brfq4ggzlq58rg9r-unit-mdadm-shutdown.service.drv'...                                                                                                                                                    
building '/nix/store/8aljm8qrg77yjm3v9ig9c5f6jklb5542-unit-mdmon-.service.drv'...                                                                                                                                                            
building '/nix/store/n6rah3sri7xhvczdvrmdnnnalkvjm5rr-unit-network-interfaces.target.drv'...                                                                                                                                                 
-- Detecting CXX compile features - done                                                                                                                                                                                                     
-- Found Antlr2: /nix/store/ksq8x1bxvnhq9amhabn4b51szf180vlc-antlr-2.7.7/lib/libantlr.a                                                                                                                                                      
-- Found QScintilla: /nix/store/5np1kx24yznzvikclkzmwda04nwx3mim-qscintilla-qt5-2.9.4/lib/libqscintilla2.so                                                                                                                                  
CMake Error at libs/qhexedit/CMakeLists.txt:25 (qt5_use_modules):                                                                                                                                                                            
  Unknown CMake command "qt5_use_modules".                                                                                                                                                                                                   




-- Configuring incomplete, errors occurred!                                                                                                                                                                                                  
See also "/tmp/nix-build-sqlitebrowser-3.10.1.drv-0/source/build/CMakeFiles/CMakeOutput.log".                                                                                                                                                
builder for '/nix/store/92xx9b3hrzgz6iq4i15ihj7mp6b5q9qb-sqlitebrowser-3.10.1.drv' failed with exit code 1                                                                                                                                   
cannot build derivation '/nix/store/yf3whjd5wqz8p6l47zmf4qm79r7zwyxi-system-path.drv': 1 dependencies couldn't be built                                                                                                                      
building '/nix/store/gh2id8g74k29vayhk2bfza3m96di8gsx-unit-network-online.target.drv'...                                                                                                                                                     
building '/nix/store/8gjnilqy9sisfcg1rdmdrh31rjfdsiwf-unit-nix-daemon.service.drv'...   
```


### Steps to reproduce



## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.


 - system: `"x86_64-linux"`                                                                                                                                                                                                                  
 - host os: `Linux 4.15.12, NixOS, 18.09pre133640.ea145b68a01 (Jellyfish)`                                                                                                                                                                   
 - multi-user?: `yes`                                                                                                                                                                                                                        
 - sandbox: `no`                                                                                                                                                                                                                             
 - version: `nix-env (Nix) 2.0`                                                                                                                                                                                                              
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`                                                                                                                                                                     
 - channels(root): `"nixos-18.09pre143628.cac9f088103"`

**Comment by manveru** [comment]
Seems related to https://github.com/sqlitebrowser/sqlitebrowser/issues/1421

**Comment by sjau** [comment]
But how to fix it?

**Comment by jgillich** [comment]
Until upstream fixes it we could probably patch around it: https://github.com/NixOS/nixpkgs/commit/05fd6a0c87b8e2dab3a36982a0ea2f06938446d6

**Comment by jgillich** [comment]
#42351 would fix it as well.

**Comment by peterhoeg** [comment]
Another option is to build with qmake instead of cmake.

**Comment by Ma27** [comment]
@peterhoeg this is basically what I just tried out. I guess that it's way easier for to use qmake, the software supports both and when I look at things like missing `qt5_use_modules` calls (there was a patch in fact, but it doesn't apply onto the latest stable release)

I'm currently testing the application, after that I'll file a patch

**Comment by orivej-nixos** [comment]
Fixed in #42351.
